rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -197,10 +209,11 @@ def _create_acl(self, lswitch, lports, acl_create_args, acls_per_port):
         ovn_nbctl.enable_batch_mode()
         for lport in lports:
             for i in range(acls_per_port):
-                match = '%s == "%s" && ip4 && udp && udp.src == %d' % \
-                        (p, lport["name"], 100 + i)
+                match = match_template % { 'direction' : p,
+                                           'lport' : lport["name"],
+                                           'address_set' : address_set,
```

Comment:
Firstly, address_set is not in the match_template.
Secondly, if user didn't supply address_set (i.e. address_set == ""), we should not put it in the generated ACL

Reply:
Do you mean in the default match_template? The default template is the one used by create_and_list_acls() or create_and_delete_acls().
Since address_set is not in the default template, it is up to the user to provide both a match_template containing an address_set reference and a valid address_set id. If the user provides a templates not containing an address_set reference, it will not be put in the ACL rule. Am I missing something?

Reply:
Now I got it. Thanks for explain. LGTM!